packages/editor/src/components/table-of-contents/style.scss

```diff
@@ -35,24 +35,21 @@
 }
 
```

Comment:
❓ Newb Question - How do I get the 'Document Outline' section to show up in this as well as is with the screenshot you shared above?

Reply:
It will show up automatically once you have some Heading blocks.

Reply:
That it does.  Thanks! 😄